plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/GcsStorageFactory.java

```diff
@@ -89,7 +86,7 @@ public Storage create(ConnectorSession session, String path)
                 }
             }
             else {
-                credential = jsonGoogleCredential.get();
+                credential = jsonGoogleCredential.orElseThrow(() -> new IllegalStateException("GCS credentials not configured"));
```

Comment:
fail fast in the constructor when the fields get initialized and not only when a connection gets created